packages/web-runtime/src/components/Account/CalDavUrl.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div v-if="checkedCalDavAvailability && isCalDavAvailable">
+    <account-table
+      :title="$gettext('Calendar')"
+      :fields="[
+        $gettext('CalDAV information name'),
+        $gettext('CalCAV information value'),
+        $gettext('CalCAV information actions')
+      ]"
+      class="account-page-caldav"
+    >
+      <template #header="{ title }">
+        <h2 class="oc-flex oc-flex-middle oc-mb-s">
+          {{ title }}
+          <oc-tag :rounded="true" color="primary" appearance="filled" size="small" class="oc-ml-s">
+            {{ $gettext('NEW') }}
+          </oc-tag>
+        </h2>
+        <span
+          class="oc-mt-l oc-text-small"
+          v-text="
+            $gettext(
+              'Here, you can access your personal calendar for integration with third-party apps like Thunderbird, Apple Calendar, and others.'
+            )
+          "
+        />
+      </template>
+
+      <oc-table-tr class="account-page-info-caldav-integration"></oc-table-tr>
+      <oc-table-tr class="account-page-info-caldav-url">
+        <oc-table-td>{{ $gettext('CalDAV URL') }}</oc-table-td>
+        <oc-table-td>
+          <span class="oc-text-truncate">{{ configStore.serverUrl }}</span>
```

Comment:
Did this change ? I thought we need a full url with /caldav slug, now its only the server url ?

Reply:
Only the server URL is correct, because calendar apps will use the .well-known endpoints automagically.